kanidm_tools/src/cli/account.rs

```diff
@@ -68,7 +72,13 @@ impl AccountOpt {
                         acsopt.aopts.account_id.as_str(),
                         password.as_str(),
                     ) {
-                        eprintln!("Error -> {:?}", e);
+                        match e {
+                            // TODO: once the password length is configurable at a system level (#498), pull from the configuration.
+                            ClientErrorHttp(_, Some(PasswordBadListed), _) => error!("Password is banned by the administrator of this system, please try a different one."),
+                            ClientErrorHttp(_, Some(PasswordTooShort(PW_MIN_LENGTH)), _) => error!("Password was too short (needs to be at least {} characters), please try again.", PW_MIN_LENGTH),
```

Comment:
We shouldn't need to match on min_length here - this is a variable, that indicates to the client what the min length is. So I think you can remove the import of PW_MIN_LEGNTH, and assign to say "min_len" and then display that. Seems simpler don't you think?

Reply:
Fixed in 35076d3, thanks :)